### DISCUSSION TITLE
websocket connect成功后，私聊机器人/echo没有反应

### DISCUSSION THREAD

**Discussion by littletiga3** [nonebot/discussions#249] [问答]
![image](https://user-images.githubusercontent.com/95532600/202847000-cb2271fe-7c5a-47ea-8da9-7548fc2b2766.png)
websocket connect成功后，私聊机器人/echo没有反应，nonebot日志里也没有表示正在处理的相关信息，go-cqhhttp日志有接收到的日志
![image](https://user-images.githubusercontent.com/95532600/202847462-3231630a-b710-4bf0-bfcc-5b1f8e57490e.png)
请问这个问题要怎么解决，相关的讨论都看过了没有结果

**Comment by he0119** [comment]
https://github.com/nonebot/nonebot2/blob/a7b0eb10a0cf7bc835c308e378dd04d28c8e26c4/nonebot/plugins/echo.py#L6

如果在群里需要@机器人的。我并没有在你的日志中看到私聊的情形。~~或者你要不要按一下 esc，你可能把命令行暂停了~~

**Reply by littletiga3** [reply]
是因为我没有办法了所以创了个群艾特它看有没有用，实际上两者都不行
![捕获](https://user-images.githubusercontent.com/95532600/202849742-a4d976a0-ff88-424c-ade1-b04ea8750c8e.PNG)

**Reply by he0119** [reply]
我发现了，你怎么用的ob12，你该用ob11适配器的。go-cqhttp目前只支持ob11。

**Reply by littletiga3** [reply]
哦哦哦换过来之后解决了，谢谢谢谢dalao，解决了我几天的困惑，今晚可以早点睡了hhh

**Comment by Well2333** [comment]
你可以测试一下其他消息nb收不收得到，或者查看nb的窗口标题栏是否有“选择”，按照你这种情况很有可能是因为nonebot的进程被挂起了

**Reply by littletiga3** [reply]
天气示例也不行，窗口标题也没有选择
![捕获](https://user-images.githubusercontent.com/95532600/202849872-70bc01b0-2a15-4e99-8cd5-b08bff4fbd18.PNG)

**Reply by Well2333** [reply]
> 天气示例也不行，窗口标题也没有选择
> ![捕获](https://user-images.githubusercontent.com/95532600/202849872-70bc01b0-2a15-4e99-8cd5-b08bff4fbd18.PNG)
> 

那尝试使用 onebot.v11 适配器试一下，可能是 v12 出现了什么问题

**Reply by yanyongyu** [reply]
gocq是v11协议。。不是v12

**Comment by Well2333** [comment]
顺便，推荐使用 [Windows Terminal](https://learn.microsoft.com/zh-cn/windows/terminal/) 来避免这种问题

**Reply by littletiga3** [reply]
试过了但好像并没有用qwq
